pybossa/importers/csv.py

```diff
@@ -87,8 +87,14 @@ def _import_csv_tasks(self, csvreader):
                                 task_data[headers[idx]] = {}
                         else:
                             task_data[headers[idx]] = cell
+                    elif headers[idx].endswith('_gold'):
+                        if 'gold_answers' not in task_data:
+                            task_data['gold_answers'] = {}
+                        task_data['gold_answers'][headers[idx].strip('_gold')] = cell
```

Comment:
`.strip` removes all the characters in the `'_gold'` string, so that's not what you want here, e.g. `'old_gold'.strip('_gold') == ''`